skymap_scanner/server/start_scan.py

```diff
@@ -259,14 +267,22 @@ def _gen_pframes(
                     time = self.nsides_dict[coarser_nside][coarser_pixel].time
                     energy = self.nsides_dict[coarser_nside][coarser_pixel].energy
 
-        for i in range(0,len(self.pos_variations)):
+        # Now generate the vertex seed position variations according to the reco-specific logic.
+
+        n_pos_variations = len(self.pos_variations)
+
+        LOGGER.debug(f"Generating {n_pos_variations} position variations.")
+
+        for i in range(0, n_pos_variations):
```

Comment:
python's `len()` is O(1), so you don't technically need `n_pos_variations`. While you're at it, `for i in range(0, n_pos_variations)` could be replaced with iterating `enumerate(self.pos_variations)`

Reply:
I just didn't like writing `len()` many times. But I am happy to switch to `enumerate` and get rid of the auxiliary variable.